.eslintrc.yaml

```diff
@@ -26,6 +26,9 @@ plugins:
   - 'react'
   - 'react-hooks'
 overrides:
+  - files: ['*.story.js']
+    rules:
+      'max-classes-per-file': 0
```

Comment:
I think it makes more sense to add an override for stories, as they had a bunch of these, and we don't really care about them as much as we do our components